.binder/environment.yml

```diff
@@ -0,0 +1,9 @@
+name: pyfstat-binder
+channels:
+  - conda-forge
+dependencies:
+  - pyfstat
+  - sphinx-gallery
+  - pip:
+      ipynbname
+
```

Comment:
From the binder console:
![binder](https://user-images.githubusercontent.com/28396587/103404425-c38b7f00-4b53-11eb-88ad-38f02584b7bb.png)
I guess this is the file where it wants that extra `pip` to go?

Reply:
Yes. To make it clear: The `- pip:` section does not include `pip` as a dependency, meaning there has to be an explicit `- pip` (no colon), allowing the version to be specified.

Also, there's a syntax error: `ipynbname` should have a hyphen!